scripts/chain/start_chain.sh

```diff
@@ -235,3 +235,53 @@ do
     echo "INFO: Checking $DAEMON_HOME-${a} chain status"
     $DAEMON status --node tcp://localhost:${RPC}
 done
+
+echo "Installing price-feeder binary"
+cd $REPO/price-feeder/
+make install
+cd $CURPATH
+
+for (( a=1; a<=$NUM_VALS; a++ ))
+do
+    DIFF=$(($a - 1))
+    INC=$(($DIFF * 2))
+    RPC=$((16657 + $INC))
+    GRPC=$((9092 + $INC))
+    PF_PORT=$((7171 + $INC))
+
+    # Copy the price-feeder config template and replace variables
+    CONFIG_DIR="${DAEMON_HOME}-${a}/config"
+    PF_CONFIG="${CONFIG_DIR}/price-feeder.toml"
+    cp ../configs/price-feeder.toml $CONFIG_DIR
+
+    UMEE_E2E_PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
```

Comment:
Let's remove the UMEE_E2E parts of these environment variables:
```suggestion
    PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
```

Reply:
Sounds good, they are a little long. I'll update them in the price-feeder config as well.